K8s-deployment/misc/rs-delete-Script/example-secrets/secrets/script-config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "dataBrokerHost" : "<databroker host>",
+    "dataBrokerPort" : 28041,
+    "dataBrokerUser" : "dx-rs-user",
+    "dataBrokerPassword" : "<rs-user-password>",
+    "dataBrokerVhost":"IUDX",
+
+    "dataBaseName":"iudx_rs",
+    "dataBaseUser":"postgres",
+    "dataBasePassword": "<database Password>",
+    "dataBaseHost":"tasks.postgres",
```

Comment:
Please add K8s specific svc addresses. Bit more specific in label placeholder values